.github/workflows/publish_pypi.yml

```diff
@@ -213,7 +247,7 @@ jobs:
   open_failure_issue:
     needs: [build_windows_wheels, build_macos_and_linux_wheels, build_sdist]
     name: Open an issue if build fails
-    if: ${{ always() && contains(needs.*.result, 'failure') && github.repository_owner == 'pybamm-team'}}
+    if: ${{ contains(needs.*.result, 'failure') && github.repository_owner == 'pybamm-team'}}
```

Comment:
Why was `always()` removed from here?

Reply:
I removed it because its presence negated the existence of the other conditions – this job ran even in forks despite not being supposed to. The check for failures in the condition seems to be good enough for this purpose.

Reply:
This would not work without `always()`. If any of the wheel jobs fail, the `open_failure_issue` job will never be executed. Using always is a bit hacky but that is what makes this work.

Maybe we can create a new issue for this and keep it as it is until we find a solution?

Reply:
Makes sense, Actions are disabled by default on forks anyway. Happy to revert the change for now